source/numerics.tex

```diff
@@ -9029,519 +9034,370 @@
+                                  @\placeholder{floating-point-type}@ z);
   constexpr float fmaf(float x, float y, float z);
   constexpr long double fmal(long double x, long double y, long double z);
 
   // \ref{c.math.lerp}, linear interpolation
-  constexpr float lerp(float a, float b, float t) noexcept;
-  constexpr double lerp(double a, double b, double t) noexcept;
-  constexpr long double lerp(long double a, long double b, long double t) noexcept;
+  constexpr @\placeholder{floating-point-type}@ lerp(@\placeholder{floating-point-type}@ a, @\placeholder{floating-point-type}@ b,
+                                    @\placeholder{floating-point-type}@ t) noexcept;
 
   // \ref{c.math.fpclass}, classification / comparison functions
-  constexpr int fpclassify(float x);
-  constexpr int fpclassify(double x);
-  constexpr int fpclassify(long double x);
-
-  constexpr bool isfinite(float x);
-  constexpr bool isfinite(double x);
-  constexpr bool isfinite(long double x);
-
-  constexpr bool isinf(float x);
-  constexpr bool isinf(double x);
-  constexpr bool isinf(long double x);
-
-  constexpr bool isnan(float x);
-  constexpr bool isnan(double x);
-  constexpr bool isnan(long double x);
-
-  constexpr bool isnormal(float x);
-  constexpr bool isnormal(double x);
-  constexpr bool isnormal(long double x);
-
-  constexpr bool signbit(float x);
-  constexpr bool signbit(double x);
-  constexpr bool signbit(long double x);
-
-  constexpr bool isgreater(float x, float y);
-  constexpr bool isgreater(double x, double y);
-  constexpr bool isgreater(long double x, long double y);
-
-  constexpr bool isgreaterequal(float x, float y);
-  constexpr bool isgreaterequal(double x, double y);
-  constexpr bool isgreaterequal(long double x, long double y);
-
-  constexpr bool isless(float x, float y);
-  constexpr bool isless(double x, double y);
-  constexpr bool isless(long double x, long double y);
-
-  constexpr bool islessequal(float x, float y);
-  constexpr bool islessequal(double x, double y);
-  constexpr bool islessequal(long double x, long double y);
-
-  constexpr bool islessgreater(float x, float y);
-  constexpr bool islessgreater(double x, double y);
-  constexpr bool islessgreater(long double x, long double y);
-
-  constexpr bool isunordered(float x, float y);
-  constexpr bool isunordered(double x, double y);
-  constexpr bool isunordered(long double x, long double y);
+  constexpr int fpclassify(@\placeholder{floating-point-type}@ x);
+  constexpr bool isfinite(@\placeholder{floating-point-type}@ x);
+  constexpr bool isinf(@\placeholder{floating-point-type}@ x);
+  constexpr bool isnan(@\placeholder{floating-point-type}@ x);
+  constexpr bool isnormal(@\placeholder{floating-point-type}@ x);
+  constexpr bool signbit(@\placeholder{floating-point-type}@ x);
+  constexpr bool isgreater(@\placeholder{floating-point-type}@ x, double y);
```

Comment:
The second parameter of `isgreater` should be `@\placeholder{floating-point-type}@ y`, not `double y`.

Reply:
> Section [complex.special] is supposed to be removed in its entirety. The explicit specializations for `std::complex` are no longer needed with the wording changes in this paper. There is an easily missed one sentence instruction to remove the section in https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2022/p1467r9.html#complex-wording

Thank you --  I _really_ wish all wording would be presented in underline/strikethrough and coloured diffs, _all_ of it. For large changes it's OK to just show a short fragment and them say "and 20 more similar cases" or use "[...]" for the middle bits, but having a visual diff marker per edit is immensely valuable.

Thanks also, Jens!